src/instruction/set.ts

```diff
@@ -117,9 +116,8 @@ export class InstructionSet implements IInstructionSet {
         return instruction
       }
     }
-    throw new VirtualBoardError(
-      `Unable to find instruction encoder for the instruction '${name}'.`,
-      VirtualBoardErrorType.NoEncoderFound
+    throw new Error(
+      `Unable to find instruction encoder for the instruction '${name}'.`
```

Comment:
IMO here an InstructionError, since user can write whatever he/she wants

Reply:
Yes you're right I will change that.